lib/rules/headers/copyright.js

```diff
@@ -42,6 +45,17 @@ exports.check = async function (sr, done) {
     if (copyright) {
         let allowBothLicense = false;
         const chartersData = await sr.getChartersData();
```

Comment:
This should be moved after the TAG/AB check. There's no need to make additional requests to get the charters if the document is produced by the TAG/AB